Cognite.Simulator.Utils/LicenseController.cs

```diff
@@ -130,6 +130,7 @@ public void AcquireLicense(CancellationToken cancellationToken) {
                     _logger.LogInformation("Attempting to acquire license");
                     _acquireLicenseFunc(cancellationToken);
                     _licenseHeld = true;
+                    _scheduledReleaseTime = _timeProvider.GetUtcNow().Add(_licenseLockTime);
```

Comment:
I think I was saying this before, but the correct place for this should be in `EndUsage` and ot here, what's the purpose of it here?

Reply:
EndUsage already has it and ignore this line, Ive removed it. It was just a local experiment accidentally committed.